src/status_im/chat/models/message.cljs

```diff
@@ -176,12 +186,13 @@
         message                       (-> raw-message
                                           (commands-receiving/enhance-receive-parameters cofx)
                                           (ensure-clock-value chat)
+                                          (check-response-to old-id->message)
                                           ;; TODO (cammellos): Refactor so it's not computed twice
                                           (add-outgoing-status current-public-key))]
     (fx/merge cofx
               {:transport/confirm-messages-processed [{:web3   web3
                                                        :js-obj js-obj}]}
-              (add-message batch? message current-chat?)
+              (add-message true message current-chat?)
```

Comment:
do we call it with `false` anywhere?

Reply:
yes, we do https://github.com/status-im/status-react/blob/812a2454aaa7d59f128b699016fdbb5c7aa103e7/src/status_im/chat/models/message.cljs#L346